nri-config-generator/templates/default/config.json.tmpl

```diff
@@ -27,6 +27,9 @@
                     "version": "{{.integration_version}}",
                     "name": "{{.integration}}"
                 },
+                {{- if .config.scrape_timeout }}
```

Comment:
which is the integration using this default? config.tmpl?

Reply:
The default is used when `exporters/<integration>/<integration>.prometheus.json.tmpl` [does not exists](https://github.com/newrelic/newrelic-prometheus-exporters-packages/blob/main/scripts/build.sh#L34-L40). Currently only `mongodb3` is defining it, so both `powerdns` and `ibmmq` use the default.

Reply:
powerdns has 
 - `nri-config-generator/integration-tests/testdata/integration_template/nri-powerdns.prometheus.json.tmpl`

Reply:
Well, it is named `powerdns` but it is only used in the integration tests, isn't it?

Reply:
oh. I am really not sure then why we use a different template in integration tests 😅 

but I guess you are 100% I did not notice the path!